content/articles/how-sweet-is-kubernetes/index.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: engineering-education
+status: publish
+published:true 
+url: /engineering-education/how-sweet-is-kubernetes/
+title: How sweet is Kubernetes?
+description: .
+author: qoyum-yusuf
+date: 2021-05-01T00:00:00-08:00
+topics: [Containers]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/quoyum-yusuf/hero.jpg
+    alt: How sweet is Kubernetes cover image 
+---
+
+
+
+
+**HOW SWEET IS KUBERNETES?**
+
+![kubernetes.jpeg](kub.jpeg)
```

Comment:
I think you can have this as your hero image.

Reply:
so you mean the image should come first right?

Reply:
Not really.
But just rename the image file to hero image.
This image will be used as the articles main image. ![Screenshot_20210530_210745_com.android.chrome.jpg](https://user-images.githubusercontent.com/45882800/120115159-1b71f480-c18b-11eb-9aa6-3393e52d0c93.jpg)

Reply:
ok thanks